Source/WebKit/UIProcess/Cocoa/WKTextExtractionItem.swift

```diff
@@ -23,8 +23,7 @@
 
 import Foundation
 
-@available(iOS 17.0, macOS 12.0, *)
-@objc(WKTextExtractionItem) public class WKTextExtractionItem: NSObject {
+@objc(WKTextExtractionItem) class WKTextExtractionItem: NSObject {
```

Comment:
is the `(WKTextExtractionItem)` part needed? (ditto for the others)

Reply:
Yes, they're required.